src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -42,9 +42,9 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig, gdDAO: Datap
 
   def createCluster(googleProject: GoogleProject, clusterName: ClusterName, clusterRequest: ClusterRequest): Future[Cluster] = {
     def create() = {
-      createGoogleCluster(googleProject, clusterName, clusterRequest) flatMap { cluster: Cluster =>
+      createGoogleCluster(googleProject, clusterName, clusterRequest) flatMap { case (cluster: Cluster, initBucket: GcsBucketName) =>
         dbRef.inTransaction { dataAccess =>
-          dataAccess.clusterQuery.save(cluster)
+          dataAccess.clusterQuery.save(cluster, GcsPath(initBucket, GcsRelativePath("")))
```

Comment:
This is fine; just wondering if `GcsBucketName` should have a `.toUri` method so you don't have to construct a `GcsPath` just to get the uri.

Reply:
yeah, or a `.toPath` or something. see my horrible test code of "just give me a GcsPath from a raw string" function, which is similar